File: Contest/Contest 8.8/Q1.cpp
#include <iostream>

int main(){
	unsigned long long input, ans;
	std::cin >> input;
	if(input % 2 == 0){
		ans = (input / 2) * (input + 1) + 1;
	}
	else{
		ans = (input + 1) / 2 * input + 1;
	}
	std::cout << ans << std::endl;

	return 0;	
}

File: Atcoder/Dwango Programming Contest V/Q1.py
N = int(input())
a = list(map(int, input().split()))

average = sum(a) / len(a)

min_diff = abs(a[0] - average)
ans = 0
for i, x in enumerate(a):
    if min_diff > abs(x - average):
        min_diff = abs(x - average)
        ans = i

print(ans)
